portfolio/src/main/webapp/art.html

```diff
@@ -0,0 +1,40 @@
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Chiamaka Chukwuka</title>
+    <link href="https://fonts.googleapis.com/css?family=Princess+Sofia" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Miss+Fajardose" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Ewert" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Libre+Barcode+39+Extended+Text" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Cinzel+Decorative" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <div id="content">
+    <div id="logo"><em><h1 id="first"><div id="lll"><span id="c1">Chi</span><span id="c2">&#10037;a<span id="c23">&#10037;</span></span><span id="c3">m</span><span id="c4">a</span><span id="c45">&#10037;</span><span id="c5">k</span><span id="c6">a</span></div></h1></em></div>
+    <h3 id="subhead"> &#9830; Chiamaka  (Chee-uh-mok-uh)  Chukwuka  (Chew-kwoo-kah). &#9830;</h3>
+    <script src="script.js"></script>
+    </div>
+    <head><title>Artwork</title></head>
+    <body>
+      <link href="https://fonts.googleapis.com/css?family=Playfair+Display+SC" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Homemade+Apple" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Monsieur+La+Doulaise" rel="stylesheet">    
+      <link href="gala.css" rel="stylesheet" type="text/css" />
+      <header>
+        <ul class="navigation">
+      <a href="./index.html"><li>HOME</li></a>
+      <a href="./projects.html"><li>PROJECTS</li></a>
+      <a href="./art.html"><li>ARTWORK</li></a>
+      <a href="./aboutme.html"><li>ABOUT ME</li></a>
+      <a href="./other.html"><li>OTHER LINKS</li></a>
+    </ul>
+      </header>
+    <div id ="art">
+    <p id = "desc"> CLICK ON PHOTOS FOR HD </p>
+      <div class="separator" style="clear: both; text-align: center;">
+<a href="https://1.bp.blogspot.com/-ZMQJzalMnd8/XP-Zsen93fI/AAAAAAAAAcE/q-8zhYvAJCYN4_zQx8cFVGgmOqHMpg2iACLcBGAs/s1600/IMG_7943.jpg" imageanchor="1" style="margin-left: 1em; margin-right: 1em;"><img border="0" data-original-height="1944" data-original-width="3600" height="729" src="https://1.bp.blogspot.com/-ZMQJzalMnd8/XP-Zsen93fI/AAAAAAAAAcE/q-8zhYvAJCYN4_zQx8cFVGgmOqHMpg2iACLcBGAs/s640/IMG_7943.jpg" width="1280" /></a></div>
```

Comment:
This file looks cut-off, you're missing at least a \</body\>.  Also, add some newlines to make this more readable

Reply:
Resolving because I added closing tags